check_test.go

```diff
@@ -48,7 +48,7 @@ func TestEncodeDomainCheck(t *testing.T) {
 func TestEncodeDomainCheckLaunchPhase(t *testing.T) {
 	con := &Conn{}
 	con.Greeting.Extensions = []string{ExtLaunch}
-	err := con.encodeDomainCheck([]string{"hello.com", "foo.domains", "xn--ninja.net"}, "claims")
+	err := con.encodeDomainCheck([]string{"hello.com", "foo.domains", "xn--ninja.net"}, map[string]string{"launch:phase": "claims"})
```

Comment:
Oops, that's only applicable when the map literal is in a slice literal. https://golang.org/doc/go1.5#map_literals